config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-6-upgrades.yaml

```diff
@@ -1,10 +1,9 @@
 periodics:
```

Comment:
Please rebase this PR against the latest master branch, so I can easily diff if we missed something